src/strings/PowerFxResources.en-US.resx

```diff
@@ -3291,21 +3291,41 @@
     <comment>function_parameter - Second argument to the Patch function - the updates to be applied to the given rows.</comment>
   </data>
   <data name="AboutRemove" xml:space="preserve">
-    <value>Removes a specific record or records from a data source</value>
+    <value>Removes (optionally All) items from the specified 'collection'.</value>
```

Comment:
>All [](http://example.com/codeflow?start=31&length=3)

`all` (lower case). We either use the proper enum name ('RemoveFlags.All') or treat 'all' as a common word.